app/scripts/models/auth_brokers/oauth.js

```diff
@@ -92,6 +95,18 @@ define([
       return p.reject(new Error('subclasses must override sendOAuthResultToRelier'));
     },
 
+    finishOAuthSignInFlow: function (account, additionalResultData) {
+      if (! additionalResultData) { additionalResultData = {}; }
```

Comment:
This is my lone nit in this PR. 

We usually set defaults/missing parameters using this form: 
```js
additionalResultData = additionalResultData || {};
```